### DISCUSSION TITLE
Question: how to build OCaml bindings on i686 where `ocamlopt` is no longer available?

### DISCUSSION THREAD

**Discussion by MIvanchev** [Z3Prover/z3#7112] [Q&A]
OCaml 5 no longer provides a native compiler for 32 bit platforms. How can I build z3 without it? Currently I get an error that `ocamlopt` is not found.

**Comment by wintersteiger** [comment]
I think the bytecode version should still work, it's just that the build scripts aren't prepared for disabling the native version of the bindings. Instead of running `make` in the build directory, try `make ml_example.byte`. If all our dependencies are set up correctly, this will build only the parts required for the bytecode example.
